resources/schema.json

```diff
@@ -467,6 +467,52 @@
         }
       }
     },
+    "scopes_old":  {
+      "$id": "#/definitions/scopes_old",
+      "type": "object",
+      "title": "element template scope",
+      "deprecated": true,
+      "description": "@Deprecated - Special scoped bindings that allow you to configure nested elements",
+      "additionalProperties": false,
+      "errorMessage": {
+        "additionalProperties": "invalid scope, object descriptor is only supported for camunda:Connector"
+      },
+      "properties": {
+        "camunda:Connector": {
+          "type": "object",
+          "title": "element template scoped connector binding",
+          "description": "A scoped binding for a camunda connector inside the element",
+          "properties": {
+            "properties": {
+              "$ref": "#/definitions/properties"
+            }
+          }
+        }
+      }
+    },
+    "scopes": {
+      "$id": "#/definitions/scopes",
+      "type": "array",
+      "description": "Special scoped bindings that allow you to configure nested elements",
```

Comment:
Any reasons why we dont have a `title` property for `scopes` (we have it for the other cases)